sdk/servicebus/azure-servicebus/azure/servicebus/_common/auto_lock_renewer.py

```diff
@@ -133,6 +133,8 @@ def register(self, renewable, timeout=300, on_lock_renew_failure=None):
 
         :rtype: None
         """
+        if not isinstance(renewable, ReceivedMessage) and not isinstance(renewable, ServiceBusSession):
+            raise TypeError("AutoLockRenewer only supports registration of types azure.servicebus.ReceivedMessage (via a receiver's receive methods) and azure.servicebus.ServiceBusSession (via a session receiver's property receiver.Session).")
```

Comment:
nit: it's "session" not "Session"

```suggestion
            raise TypeError("AutoLockRenewer only supports registration of types azure.servicebus.ReceivedMessage (via a receiver's receive methods) and azure.servicebus.ServiceBusSession (via a session receiver's property receiver.session).")
```